actions/install/action.yml

```diff
@@ -12,7 +12,7 @@ inputs:
   local:
     description: Build and use the local version of the Catalyst-CI CLI
     required: false
-    default: 'false'
+    default: 'true'
```

Comment:
Reminder: Once everything is done and ready to merge, change back to 'false'

Reply:
yup changed back to 'false'